ls8/cpu.py

```diff
@@ -62,4 +75,36 @@ def trace(self):
 
     def run(self):
         """Run the CPU."""
-        pass
+
+        #check for halted
+        halted = False
+        # run while loop and conditionals
+        while not halted:
+            instruction = self.ram[self.pc]
+
+            if instruction == 0b10000010: # LDI
```

Comment:
How can we improve readability here? Perhaps mapping the binary value to the commands?